Siva - Fix Dates and Online Filters” Misalignment

pavanputti opened the pull request:
# Description
<img width="601" height="431" alt="Screenshot 2025-11-16 at 1 33 32 AM" src="https://github.com/user-attachments/assets/1617277b-d91f-4ca9-b934-c6c2c87fb683" />
<img width="548" height="669" alt="Screenshot 2025-11-16 at 1 34 33 AM" src="https://github.com/user-attachments/assets/76039e85-a319-45f4-941f-e1a012ead6fd" />

## Related PRS (if any):
This frontend PR is related to the development backend PR. To test this frontend PR, you need to checkout the development backend PR.
…

## Main changes explained:
- Delete file A for removing unused components …
- Update file B for including new pattern …
- Create file C for introducing new components …
…

## How to test:

1. Check out branch: siva/fix-filter-alignment-figma-match-clean
2. Run npm install (if needed)
3. Start the development server
4. Navigate to: /communityportal/
5. Open the Search Filters panel on the left sidebar
6. Verify:
- "Dates" label is properly aligned
- "Tomorrow" and "This Weekend" radio buttons are aligned with consistent indentation
- "Online" label is properly aligned
- "Online Only" checkbox is aligned with consistent indentation
- All filter elements match the Figma design spacing and alignment

## Screenshots or videos of changes:
<img width="1440" height="801" alt="Screenshot 2025-11-16 at 1 24 46 AM" src="https://github.com/user-attachments/assets/5cfdc67d-26b2-4510-afef-9807da0c58b3" />

pavanputti committed:
Set up CSS structure for filter alignment to match Figma design

pavanputti committed:
Set up CSS structure for filter alignment to match Figma design

pavanputti committed:
Align filter options with consistent indentation to match Figma design

pavanputti committed:
Merge commit 2: Align filter options with consistent indentation to match Figma design

pavanputti committed:
Refine typography and spacing consistency for filter alignment

- Added CSS variables for font sizes and line heights
- Standardized typography across filter labels and options
- Improved visual hierarchy with consistent font sizing
- Enhanced spacing alignment with refined margin-top values
- Applied consistent letter-spacing and line-height for better readability

pavanputti committed:
Add final polish and interactive states for filter alignment

- Updated date input to use CSS module class for consistent styling
- Added hover states for all interactive filter elements
- Enhanced visual feedback with accent-color for radio/checkbox inputs
- Added box-sizing border-box for consistent sizing
- Improved label hover states for better user experience
- Ensured all filter inputs have consistent styling and behavior

pavanputti committed:
Fix filter alignment to match Figma design specifications

- Replaced form labels with divs for section headers (Dates, Online)
- Improved vertical alignment of radio buttons and checkboxes with labels
- Added white-space: nowrap to prevent text wrapping
- Enhanced alignment consistency with vertical-align: middle
- Fixed checkbox label structure for proper semantic HTML
- Refined spacing variables for precise Figma matching

pavanputti committed:
Merge development into siva/fix-filter-alignment-figma-match-clean, resolve conflicts in CPDashboard

HemanthNidamanuru reviewed (changes requested):
Hi,

I tested this PR locally. The radio buttons and checkbox should be on the left side of the labels, and the “Online Only” text is not aligned in the same row as the checkbox. Could you please check this?

<img width="1887" height="902" alt="Image" src="https://github.com/user-attachments/assets/68338ccb-7cd5-4533-8d4b-b3f38dbf0b7b" />

Anusha-Gali reviewed (changes requested):
Hi Siva,

I have reviewed your PR locally and mentioned by the other reviewer "Online Only" is not aligned as per requirement, else the application UI is as per requirement.

<img width="1279" height="759" alt="Image" src="https://github.com/user-attachments/assets/742f087c-a13d-48fc-82eb-4ecf6777fda9" />
<img width="1279" height="759" alt="Image" src="https://github.com/user-attachments/assets/915e88dd-84e4-4d8f-ae99-0598db4c0bc0" />

pavanputti committed:
Merge remote-tracking branch 'origin/development' into siva/fix-filter-alignment-figma-match-clean

pavanputti committed:
Revert "Merge remote-tracking branch 'origin/development' into siva/fix-filter-alignment-figma-match-clean"

This reverts commit d4ab1d7ab607253d8cfa2acd5674d18875899d29, reversing
changes made to fd567a49838d412b2d1aa48812fb47f8bb2fe659.

rohanrastogi311 reviewed (dismissed):
Hi Siva, 

Well done with this implementation. 

<img width="1728" height="1042" alt="PR 4388 Screenshot" src="https://github.com/user-attachments/assets/01b6d59a-6ba2-468b-ba39-3e2a52044edc" />

pavanputti committed:
Fix filter alignment: radio/checkbox left of labels, prevent overlap

- Move Dates/Online section headers to divs; align radio/checkbox with labels
- Give radio and checkbox explicit size (18px) and gap to prevent overlap
- Override Bootstrap form-check positioning so flex layout controls spacing
- Online Only uses FormGroup+Label for consistent alignment with date options

pavanputti committed:
Merge development into siva/fix-filter-alignment-figma-match-clean

Resolved conflicts:
- CPDashboard.jsx: removed duplicate Show Past Events block (keep header button only)
- CPDashboard.module.css: kept filter alignment/overlap fix, merged development styles, removed duplicate selectors
- ActivityComments.jsx: use feedbackSearchInput class for feedback search input

pavanputti committed:
Fix Online Only alignment: keep label on same row as checkbox

- Override display: inline for filter option labels so they stay beside control
- Add flex-wrap: nowrap to checkboxGroup to prevent wrapping

Vishnu10-hub reviewed (approved):
Hi Siva,

Checked the PR locally, everything looks correct and well-implemented.


<img width="1440" height="882" alt="Image" src="https://github.com/user-attachments/assets/fe333395-5c90-4036-bf41-67260d44012b" />

pavanputti committed:
Merge remote-tracking branch 'origin/development' into siva/fix-filter-alignment-figma-match-clean

# Conflicts:
#	package.json
#	public/index.css
#	src/actions/bmdashboard/equipmentActions.js
#	src/components/ApplicantVolunteerRatio/ApplicantVolunteerRatio.jsx
#	src/components/BMDashboard/Equipment/DailyActivityLog/EDailyActivityLog.jsx
#	src/components/BMDashboard/Equipment/DailyActivityLog/EDailyActivityLog.module.css
#	src/components/BMDashboard/Tools/SimpleToolChart.jsx
#	src/components/BMDashboard/WeeklyProjectSummary/MostFrequentKeywords/MostFrequentKeywords.jsx
#	src/components/BMDashboard/WeeklyProjectSummary/MostFrequentKeywords/MostFrequentKeywords.module.css
#	src/components/BMDashboard/WeeklyProjectSummary/WeeklyProjectSummary.jsx
#	src/components/BMDashboard/WeeklyProjectSummary/WeeklyProjectSummary.module.css
#	src/components/Collaboration/Collaboration.jsx
#	src/components/Collaboration/Collaboration.module.css
#	src/components/Collaboration/JobFormbuilder.jsx
#	src/components/Collaboration/QuestionSetManager.jsx
#	src/components/CommunityPortal/Activities/ActivityList.jsx
#	src/components/CommunityPortal/Activities/ActivityList.module.css
#	src/components/CommunityPortal/Activities/activityId/ResourcesUsage.jsx
#	src/components/CommunityPortal/CPDashboard.module.css
#	src/components/CommunityPortal/EventPersonalization/EventStats.jsx
#	src/components/CommunityPortal/EventPersonalization/EventStats.module.css
#	src/components/HGNHelpSkillsDashboard/CommunityMembersPage.jsx
#	src/components/HGNHelpSkillsDashboard/TeamCard/TeamCard.jsx
#	src/components/HGNHelpSkillsDashboard/TeamCard/TeamCard.module.css
#	src/components/HGNHelpSkillsDashboard/TeamCard/TeamMemberRow.jsx
#	src/components/HGNSkillsDashboard/SkillsProfilePage/components/ProfileDetails.jsx
#	src/components/HGNSkillsDashboard/SkillsProfilePage/styles/Skills.module.css
#	src/components/LBDashboard/Messaging/LBMessaging.module.css
#	src/components/PRGradingScreen/PRGradingScreen.jsx
#	src/components/PRGradingScreen/PRGradingScreen.module.css
#	src/components/Projects/WBS/SingleTask/SingleTask.jsx
#	src/components/TeamMemberTasks/TeamMemberTask.jsx
#	src/components/TeamMemberTasks/style.module.css
#	src/components/UserProfile/TeamsAndProjects/AddProjectPopup.jsx
#	src/components/UserProfile/TeamsAndProjects/AddTeamPopup.jsx
#	src/components/WeeklySummariesReport/WeeklySummariesReport.jsx
#	src/components/WeeklySummariesReport/WeeklySummariesReport.module.css
#	src/routes.jsx
#	src/utils/URL.js
#	yarn.lock

pavanputti committed:
chore: revert unintended files to match development

pavanputti committed:
fix(CPDashboard): resolve Dates and Online filter misaligned Figma UI mismatch

pavanputti committed:
fix(CPDashboard): remove duplicate .dateFilter selector satisfying SonarQube

one-community commented:
Thank you all, merging!
